doc/Language/objects.pod6

```diff
@@ -435,7 +432,7 @@ C<Mu.new> calls method L<bless> on its invocant, passing all the named
 arguments. C<bless> creates the new object and then calls method C<BUILDALL>
 on it. C<BUILDALL> walks all subclasses in reverse method resolution order
 (i.e. from L<Mu> to most derived classes) and in each class checks for
-existence of a method named C<BUILD>. If it exists, it is called, again
+existence of a method named C<BUILD>. If it exists, it's called, again
```

Comment:
So:

C<Mu.new> calls method L<bless> on its invocant, passing all the named
arguments. C<bless> creates the new object and then calls method C<BUILDALL>
on it. C<BUILDALL> walks all subclasses in reverse method resolution order
(i.e. from L<Mu> to most derived classes) **and in each class checks for the
existence of a method named C<BUILD>. If the method exists, the method is called with all the named arguments from the C<new> method.**